custom_components/fullykiosk/sensor.py

```diff
@@ -13,6 +13,7 @@
     "batteryLevel": "Battery Level",
     "screenOrientation": "Screen Orientation",
     "foregroundApp": "Foreground App",
+    "backgroundApp": "Background App",
```

Comment:
This isn't necessary as Fully doesn't expose a list of background apps

Reply:
okay, i remove the line